### DISCUSSION TITLE
`隐藏视频推荐`支持稍后再看和收藏播放页面，并隐藏播放器中的推荐视频

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#4174] [功能建议 / Ideas]
不太清楚以前是否支持就先不开 issue 了。
希望`隐藏视频推荐`可以：
1. 支持稍后再看和收藏播放页面
2. 隐藏视频播放完后播放器中显示的推荐视频。

既然都开了这个功能，应该是有不希望在当前页面看到任何推荐视频的需求的。

**Comment by the1812** [comment]
稍后再看和收藏播放页面是什么情况会看到播放完后的推荐视频来着, 我感觉以前会看到, 但现在都会连播下去了

**Reply by Geeyun-JY3** [reply]
啊我原意就是在普通视频播放页隐藏视频播放完后播放器中显示的推荐视频（没验证过在稍后再看和收藏播放页面放完列表最后一个视频后播放器中是否会出现推荐视频）。
支持稍后再看和收藏播放页面，隐藏播放器中的推荐视频，这是两个没啥关系的需求。
我再改改描述吧。
